downstream/assemblies/platform/assembly-aap-migration.adoc

```diff
@@ -10,11 +10,11 @@ ifdef::context[:parent-context: {context}]
 
 Migrating your {PlatformName} deployment to the {OperatorPlatform} allows you to take advantage of the benefits provided by a Kubernetes native operator, including simplified upgrades and full lifecycle support for your {PlatformName} deployments.
 
-This guide allows you to migrate:
-
-* A VM-based installation of {ControllerName} or {HubName}, or
-* An Openshift instance of Ansible Tower 3.8.6 ({PlatformNameShort} 1.2)
+Use these procedures to migrate any of the following deployments to the {OperatorPlatform}:
 
+* A VM-based installation of {ControllerName} or {HubName},
+* An Openshift instance of Ansible Tower 3.8.6 ({PlatformNameShort} 1.2), or
```

Comment:
Is it still Ansible Tower in this version?

Reply:
@ianf77 Yes, because this information is intended to help users migrate from old deployments to the new AAP Operator.